### ISSUE TITLE
Update esifycss

### ISSUE THREAD

**Issue by TheGreatRefrigerator** [kei-ito/rollup-plugin-embed-css#122]
Sorry to bother you again...

With the newest release of esifycss `npm run build:tsc` will fail with :
```
> rollup-plugin-embed-css@1.0.16 build:tsc /Users/amandusbutzer/Workspaces/rollup-plugin-embed-css
> tsc

node_modules/esifycss/lib/minifier/ast.d.ts:2:10 - error TS2305: Module '"../../../../../../../../Users/amandusbutzer/Workspaces/rollup-plugin-embed-css/node_modules/esifycss/lib/minifier/walker"' has no exported member 'IProgram'.

2 import { IProgram, IObjectExpression, IArrayExpression } from './walker';
           ~~~~~~~~

node_modules/esifycss/lib/minifier/ast.d.ts:2:20 - error TS2305: Module '"../../../../../../../../Users/amandusbutzer/Workspaces/rollup-plugin-embed-css/node_modules/esifycss/lib/minifier/walker"' has no exported member 'IObjectExpression'.

2 import { IProgram, IObjectExpression, IArrayExpression } from './walker';
                     ~~~~~~~~~~~~~~~~~

node_modules/esifycss/lib/minifier/ast.d.ts:2:39 - error TS2305: Module '"../../../../../../../../Users/amandusbutzer/Workspaces/rollup-plugin-embed-css/node_modules/esifycss/lib/minifier/walker"' has no exported member 'IArrayExpression'.

2 import { IProgram, IObjectExpression, IArrayExpression } from './walker';
                                        ~~~~~~~~~~~~~~~~

node_modules/esifycss/lib/minifier/extractCSSFromArrayExpression.d.ts:1:10 - error TS2305: Module '"../../../../../../../../Users/amandusbutzer/Workspaces/rollup-plugin-embed-css/node_modules/esifycss/lib/minifier/walker"' has no exported member 'INode'.

1 import { INode } from './walker';
           ~~~~~

node_modules/esifycss/lib/minifier/findAddStyleImport.d.ts:2:10 - error TS2305: Module '"../../../../../../../../Users/amandusbutzer/Workspaces/rollup-plugin-embed-css/node_modules/esifycss/lib/minifier/walker"' has no exported member 'IImportDeclaration'.

2 import { IImportDeclaration, IVariableDeclaration, IFunctionDeclaration } from './walker';
           ~~~~~~~~~~~~~~~~~~

node_modules/esifycss/lib/minifier/findAddStyleImport.d.ts:2:30 - error TS2305: Module '"../../../../../../../../Users/amandusbutzer/Workspaces/rollup-plugin-embed-css/node_modules/esifycss/lib/minifier/walker"' has no exported member 'IVariableDeclaration'.

2 import { IImportDeclaration, IVariableDeclaration, IFunctionDeclaration } from './walker';
                               ~~~~~~~~~~~~~~~~~~~~

node_modules/esifycss/lib/minifier/findAddStyleImport.d.ts:2:52 - error TS2305: Module '"../../../../../../../../Users/amandusbutzer/Workspaces/rollup-plugin-embed-css/node_modules/esifycss/lib/minifier/walker"' has no exported member 'IFunctionDeclaration'.

2 import { IImportDeclaration, IVariableDeclaration, IFunctionDeclaration } from './walker';
                                                     ~~~~~~~~~~~~~~~~~~~~

src/cssPlugin.ts:30:31 - error TS2339: Property 'addStyle' does not exist on type 'IParseResult'.

30             chunk.code = [css.addStyle, ...css.statements]
                                 ~~~~~~~~

src/parseBundle.ts:39:17 - error TS2345: Argument of type '{ code: string; cssKey: string; helper: string; }' is not assignable to parameter of type '{ code: string; cssKey: string; ecmaVersion?: 10 | 9 | 8 | 7 | 6 | 5 | 3 | 11 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020 | undefined; }'.
  Object literal may only specify known properties, and 'helper' does not exist in type '{ code: string; cssKey: string; ecmaVersion?: 10 | 9 | 8 | 7 | 6 | 5 | 3 | 11 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020 | undefined; }'.

39                 helper: path.relative(directory, props.helper),
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 9 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! rollup-plugin-embed-css@1.0.16 build:tsc: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the rollup-plugin-embed-css@1.0.16 build:tsc script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/amandusbutzer/.npm/_logs/2020-06-25T13_12_47_937Z-debug.log

```

I was trying to look into it, but am not familiar with TS yet, so i can't be of much help here 😕

**Comment by kei-ito** [comment]
Hi, thank you for the information. I'm now working on it and may be done in few hours.
Sorry for taking up your time.

**Comment by TheGreatRefrigerator** [comment]
> Sorry for taking up your time.
likewise 😅 

It's not urgent.
I'm already happy that i get fast feedback :)
That's not too common.

Thanks for taking care of this 👍

**Comment by kei-ito** [comment]
I published v1.0.17 that includes the latest esifycss.
